CONTRIBUTING.md

```diff
@@ -32,11 +32,12 @@ in other communication means.
 * Questions and community support that really don't fit in other
 communication means.
 
-### IRC channel
+### Slack
 
-Channel: `#grimoirelab` at [Freenode](http://freenode.net/)
+Channel: [`#grimoirelab`](https://chaoss-workspace.slack.com/archives/C022NPTPC8Z) 
```

Comment:
This link doesn't load in my browser. It doesn't redirects.

Reply:
Strange, it works well in my browser. Also, redirects to the channel and opens the slack app too. :thinking: 
Anyway, I will check it tomorrow again.

Reply:
I tried again and I need to click on a link that it's on the page that I get. It's ok for me.